src/app/components/popup-choose-the-time/popup-choose-the-time.component.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Component,
+  Input,
+} from '@angular/core';
+
+@Component({
+  selector: 'app-popup-choose-the-time',
+  templateUrl: './popup-choose-the-time.component.html',
+  styleUrls: ['./popup-choose-the-time.component.scss'],
+})
+export class PopupChooseTheTimeComponent {
+  @Input() title: string = 'Choose the time';
+  @Input() titleData: string = '01/11/2021';
+
+  public times: any = [0];
+
+  // eslint-disable-next-line
```

Comment:
Eslint should be enabled and all types returned by functions should be in place.
This one should return `void`.

Reply:
I fixed it